### DISCUSSION TITLE
Sudden page move up!

### DISCUSSION THREAD

**Discussion by Spok1** [community/community#74991] [Mobile]
### Select Topic Area

Bug

### Body

When open the menu, the page scrolls up!

https://github.com/community/community/assets/34476664/5c0fdf06-5ba3-40dd-bf1b-47bdfa2cb0c4

**Comment by iskae** [comment]
Hi @Spok1 👋 

Thanks for reaching out! It looks like your request is related to our Webpage though and the `Mobile` category is used only for our native mobile apps 😅 

Please let us know if you have any requests regarding our native apps 🙇 

Cheers!

**Reply by Spok1** [reply]
The question is only one, will it be corrected soon?
